### PR TITLE
Add test for coinsplit

### PR DESCRIPTION
**Pull request by lmoe**
# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the [contribution guidelines](/CONTRIBUTING.md) for this project
- [ ] I have performed a self-review of my own code
- [ ] I have selected the `develop` branch as the target branch
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/vm/vmtxbuilder/txbuilder_test.go [hunk 1]
```diff
@@ -81,6 +82,7 @@ func TestTxBuilderBasic(t *testing.T) {
 	)
 
 	require.NoError(t, err)
+	fmt.Print(txnResponse.Digest.String())
```
Comment (howjmay) [inline packages/vm/vmtxbuilder/txbuilder_test.go:85]: maybe remove this?

File: packages/vm/vmtxbuilder/txbuilder_test.go [hunk 2]
```diff
@@ -107,7 +109,7 @@ func TestTxBuilderSendAssetsAndRequest(t *testing.T) {
 			ChainOwnerAddress: chainSigner.Address(),
 			PackageID:         iscPackage,
 			StateMetadata:     []byte{1, 2, 3, 4},
-			InitCoinRef:       getCoinsRes.Data[1].Ref(),
```
Comment (howjmay) [inline packages/vm/vmtxbuilder/txbuilder_test.go:110]: As I mentioned in https://github.com/iotaledger/isc-private/pull/433#discussion_r1886616839
I think we should temporarily not change this

File: packages/solo/solo.go [hunk 3]
```diff
@@ -674,12 +675,17 @@ func (env *Solo) executePTB(
...
+				ShowObjectChanges:  true,
+				ShowEvents:         true,
+				ShowInput:          true,
+				ShowBalanceChanges: true,
+				ShowRawEffects:     true,
+				ShowRawInput:       true,
 			},
 		},
 	)
+	fmt.Print(execRes.Digest.String())
```
Comment (howjmay) [inline packages/solo/solo.go:688]: maybe remove this change

**Review by howjmay — CHANGES_REQUESTED** [review]

File: packages/solo/solo.go (hunk 3 above)
Comment (lmoe) [reply packages/solo/solo.go:688]: will be fixed in the golint branch

File: packages/vm/vmtxbuilder/txbuilder_test.go (hunk 1 above)
Comment (lmoe) [reply packages/vm/vmtxbuilder/txbuilder_test.go:85]: will be fixed in the golint branch

**Pull request merged**